### ISSUE TITLE
Multiple ideas from podcast-ext project

### ISSUE THREAD

**Issue by PofMagicfingers** [Podcastindex-org/podcast-namespace#173]
Hey !

I'm co-founder of podCloud, a French podcasting platform (hosting, discovering, listening platform).

We were actually working on the same project as yours, beginning months ago since 2019.

[podCloud/podcast-ext](https://github.com/podCloud/podcast-ext)

We had multiple emotions when we discovered your project. We first ignored it, then we changed our namespace to podext to prevent future issues.

Now, given the growing community supporting your initiative, I'm wondering if it's better for us to retire our project and join yours, for the better good of podcast feeds and work together.

Nevertheless we had given many thoughs on this so I guess we could keep some ideas and also propose them here.

I'm mostly thinking about episodes and [podcast licence](https://github.com/podCloud/podcast-ext#episode-metadata-tags) and our "cultural" tags, allowing podcasters to tag in episodes books, games, movies, events, TV shows, even objects. I'll let you take a look at it, let me know what you think of them.
[podCloud/podcast-ext](https://github.com/podCloud/podcast-ext#episode-metadata-tags)

**Comment by jamescridland** [comment]
This all looks super-exciting - thanks so much for sharing it.

The 'licence' looks very interesting. I wonder whether there's a method of incorporating that somehow into the <copyright> tag that already exists in RSS? (Is it possible to use a namespace to modify an existing tag?) I'm thinking of the type:

`<copyright licence="CC-BY">My Podcast Company</copyright>` 

The episode metadata is fascinating; the idea of tagging books, games and movies etc is really interesting. I wonder how this can be made webscale, and have unambiguous data about each thing? A UAN/ISBN code?

I think it definitely worth proposing some here.

**Comment by daveajones** [comment]
@PofMagicfingers This is great!  Thanks for bringing your project knowledge here.  Please add yourself to the contributors.txt file right away.  Looking through your repo, there is so much overlap of ideas. ❤️

I agree with James that the metadata stuff looks awesome. Please do some tag proposals and we can all start refining things.

**Comment by PofMagicfingers** [comment]
Thanks for the warm welcome ! I'll create issues for tag proposals in the upcoming days.

@daveajones Thanks a lot, I'll add myself ! Do you mind if we also add @Bigaston who helped me a lot on our original ideas ? :slightly_smiling_face: 

@jamescridland That's an interesting idea, I think we could do that, but technically it should be in the following form :
`<copyright podcast:licence="CC-BY">My Podcast Company</copyright>`

However I'm not sure the `copyright` tag is available at item level. We'll discuss this in the dedicated issue when I'll create it :D

**Comment by daveajones** [comment]
Yes, please add whoever has contributed in even the slightest way. Even if it’s just inspirational.  Everyone gets credit here. 😊

DJ

**Comment by PofMagicfingers** [comment]
I created all relevant issues, I guess. 

Before closing this one, we own [podcast-ext.org](podcast-ext.org) domain name, do you want to discuss what we could do with it together ?

We could redirect it to this project, or let it die.

**Comment by daveajones** [comment]
Whatever you think on the domain name. Totally up to you.

Thanks for bringing over these ideas. I’ll be updating the proposal list in the readme in the next week or so to prepare for phase 3.

**Comment by Marzal** [comment]
Hi, as per https://github.com/Podcastindex-org/podcast-namespace/discussions/749 and https://github.com/Podcastindex-org/podcast-namespace/discussions/695 we are closing old issues.

Thanks for you contribution and your open [source] views.

Please open a new [Discussions](https://github.com/Podcastindex-org/podcast-namespace/discussions) if you think that the matter it's worth pursuing.
